### ISSUE TITLE
Analytical Table: Unable to click components inside cells

### ISSUE THREAD

**Issue by agauyeung** [SAP/ui5-webcomponents-react#428]
**Describe the bug**
I have a ui5-link and a ui5-button inside a cell in Analytical Table (Multi-Select mode). I have click event listeners for both, but the handlers are not invoked when clicking either.

**Expected behavior**
Clicking these controls should dispatch a click event.

**Actual behavior**
Clicking the controls actually select/un-select the row.

**UI5 Web Components for React Information**
Operating System:
`@ui5/webcomponents-react` version:
Do you have a dependency to`@ui5/webcomponents` in your package.json? If yes, which version?:
0.9.0-rc.9

**Comment by MarcusNotheis** [comment]
Released in `0.9.0-rc.15`

**Comment by agauyeung** [comment]
Hi @MarcusNotheis,

I took a look at your fix, which is checking based on a set of components.
Unfortunately, it doesn't seem to work when the components are wrapped in another component.

In my case, I have created a cell web component (off of UI5Element) which has a collection of interactive controls. This particular cell has a ui5-link and a ui5-button. I set a breakpoint at `tagNamesWhichShouldNotSelectARow.has(e.target.tagName)` and the tagName is the name of my web component, not the ui5-link that I had actually clicked. A click event from my cell was somehow fired and caught by this code (I haven't actually defined a click event for my cell, so this is surprising.)

My cell component is listening for a click event from the ui5-link and would in turn dispatch a custom event, but the cell's ui5-link click event handler isn't even being called. This implies to me that the event is being suppressed somehow.

Hoping you might have some insight.

Thanks for the quick turnaround with this fix!
